### ISSUE TITLE
Search results return same package multiple times - one for each distro

### ISSUE THREAD

**Issue by davetcoleman** [rosindex/rosindex.github.io#68]
Is this a good design decision? There is probably some good reason I'm missing, but I would prefer to just see one instance of every package that matches my query

**Comment by jbohren** [comment]
Yeah, the issue that a search hit might be for a specific distro / fork only. There should be a way to clean that up though, so that it only shows a given package once.
